<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>297</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="im.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="im.qrc">:/images/back1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="im.qrc">:/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    
    color: rgb(255, 255, 255);                   /* رنگ متن خاکستری تیره */
    
    border-radius: 10px;           /* گوشه‌های گرد */
    
    font-size: 16px;               /* اندازه فونت */
    font-weight: bold;             /* وزن فونت */
    text-align: center;            /* تراز متن */
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);  /* سایه متن */
    
}
</string>
    </property>
    <property name="text">
     <string>To Do</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="loginwidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 100);       /* پس‌زمینه سفید */
    color: #333;                   /* رنگ متن خاکستری تیره */
    
    border-radius: 10px;           /* گوشه‌های گرد */

}</string>
    </property>
    <widget class="QLineEdit" name="PasswordInLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Base style for QLineEdit */
QLineEdit {
    padding: 10px 15px; /* Adds internal padding for a comfortable text area */
    font-size: 16px; /* Sets the font size of the text */
    border-radius: 10px; /* Rounds the corners of the QLineEdit */
    border: 0.5px solid rgba(0, 0, 0, 100); /* Sets an initial transparent border */
    background-color: rgba(255, 255, 255, 150); /* Sets the background color to white */
    background-clip: padding-box; /* Ensures background color stays within padding */
    background-origin: padding-box; /* Ensures background origin starts within padding */
    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow for a floating effect */
    transition: all 0.3s ease; /* Smooth transition for all properties */
}

/* Style for QLineEdit when it is focused */
QLineEdit:focus {
    border: 3px solid #ffffff; /* Changes border color to light blue on focus */
    box-shadow: 0 4px 10px rgba(0, 191, 255, 0.3); /* Enhances shadow to give a glowing effect on focus */
    background-color: rgba(255, 255, 255, 200); /* Changes background color to a very light blue on focus */
}

/* Style for the placeholder text inside QLineEdit */
QLineEdit::placeholder {
    color: #aaa; /* Sets the placeholder text color to light gray */
    opacity: 0.8; /* Makes the placeholder text slightly transparent */
}
</string>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBotton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:rgb(71, 135, 170); /* Sets the background color */
    color: white; /* Sets the text color */
    border-radius: 20px; /* Rounds the corners */
    padding: 9px; /* Adds padding */
    font-size: 16px; /* Sets the font size */
    border: 1px solid; /* Adds a border */
}

QPushButton:hover {
    background-color: #A90707; /* Changes background color on hover */
}
</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameInLogIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Base style for QLineEdit */
QLineEdit {
    padding: 10px 15px; /* Adds internal padding for a comfortable text area */
    font-size: 16px; /* Sets the font size of the text */
    border-radius: 10px; /* Rounds the corners of the QLineEdit */
    border: 0.5px solid rgba(0, 0, 0, 100); /* Sets an initial transparent border */
    background-color: rgba(255, 255, 255, 150); /* Sets the background color to white */
    background-clip: padding-box; /* Ensures background color stays within padding */
    background-origin: padding-box; /* Ensures background origin starts within padding */
    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow for a floating effect */
    transition: all 0.3s ease; /* Smooth transition for all properties */
}

/* Style for QLineEdit when it is focused */
QLineEdit:focus {
    border: 3px solid #ffffff; /* Changes border color to light blue on focus */
    box-shadow: 0 4px 10px rgba(0, 191, 255, 0.3); /* Enhances shadow to give a glowing effect on focus */
    background-color: rgba(255, 255, 255, 200); /* Changes background color to a very light blue on focus */
}

/* Style for the placeholder text inside QLineEdit */
QLineEdit::placeholder {
    color: #aaa; /* Sets the placeholder text color to light gray */
    opacity: 0.8; /* Makes the placeholder text slightly transparent */
}
</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: transparent; /* Makes the background transparent */
    color: #ffffff; /* Sets the text color */
    font-size: 12px; /* Sets the font size */
    font-weight: bold; /* Makes the font bold */
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow to the text */
    border: none; /* Ensures there is no border */
}
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: transparent; /* Makes the background transparent */
    color: #ffffff; /* Sets the text color */
    font-size: 12px; /* Sets the font size */
    font-weight: bold; /* Makes the font bold */
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow to the text */
    border: none; /* Ensures there is no border */
}
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpInLogIn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>60</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent; /* پس‌زمینه شفاف */
    color: rgb(0, 0, 0); /* رنگ متن اصلی */
    font-size: 10px; /* اندازه قلم */
    font-weight: bold; /* وزن قلم */
    border: none; /* بدون حاشیه */
    text-decoration: none; /* بدون تزئین متن */
    outline: none; /* بدون خط حاشیه دور روشن شدن */
    border-bottom: 2px solid black; /* خط زیری با رنگ سفید */
}

QPushButton::hover 
{
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signupwidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 100);       /* پس‌زمینه سفید */
    color: #333;                   /* رنگ متن خاکستری تیره */
    
    border-radius: 10px;           /* گوشه‌های گرد */

}</string>
    </property>
    <widget class="QLineEdit" name="PasswordInSignup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Base style for QLineEdit */
QLineEdit {
    padding: 10px 15px; /* Adds internal padding for a comfortable text area */
    font-size: 16px; /* Sets the font size of the text */
    border-radius: 10px; /* Rounds the corners of the QLineEdit */
    border: 0.5px solid rgba(0, 0, 0, 100); /* Sets an initial transparent border */
    background-color: rgba(255, 255, 255, 150); /* Sets the background color to white */
    background-clip: padding-box; /* Ensures background color stays within padding */
    background-origin: padding-box; /* Ensures background origin starts within padding */
    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow for a floating effect */
    transition: all 0.3s ease; /* Smooth transition for all properties */
}

/* Style for QLineEdit when it is focused */
QLineEdit:focus {
    border: 3px solid #ffffff; /* Changes border color to light blue on focus */
    box-shadow: 0 4px 10px rgba(0, 191, 255, 0.3); /* Enhances shadow to give a glowing effect on focus */
    background-color: rgba(255, 255, 255, 200); /* Changes background color to a very light blue on focus */
}

/* Style for the placeholder text inside QLineEdit */
QLineEdit::placeholder {
    color: #aaa; /* Sets the placeholder text color to light gray */
    opacity: 0.8; /* Makes the placeholder text slightly transparent */
}
</string>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signupBotton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:rgb(71, 135, 170); /* Sets the background color */
    color: white; /* Sets the text color */
    border-radius: 20px; /* Rounds the corners */
    padding: 9px; /* Adds padding */
    font-size: 14px; /* Sets the font size */
    border: 1px solid; /* Adds a border */
}

QPushButton:hover {
    background-color: #A90707; /* Changes background color on hover */
}
</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameInSignup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Base style for QLineEdit */
QLineEdit {
    padding: 10px 15px; /* Adds internal padding for a comfortable text area */
    font-size: 16px; /* Sets the font size of the text */
    border-radius: 10px; /* Rounds the corners of the QLineEdit */
    border: 0.5px solid rgba(0, 0, 0, 100); /* Sets an initial transparent border */
    background-color: rgba(255, 255, 255, 150); /* Sets the background color to white */
    background-clip: padding-box; /* Ensures background color stays within padding */
    background-origin: padding-box; /* Ensures background origin starts within padding */
    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow for a floating effect */
    transition: all 0.3s ease; /* Smooth transition for all properties */
}

/* Style for QLineEdit when it is focused */
QLineEdit:focus {
    border: 3px solid #ffffff; /* Changes border color to light blue on focus */
    box-shadow: 0 4px 10px rgba(0, 191, 255, 0.3); /* Enhances shadow to give a glowing effect on focus */
    background-color: rgba(255, 255, 255, 200); /* Changes background color to a very light blue on focus */
}

/* Style for the placeholder text inside QLineEdit */
QLineEdit::placeholder {
    color: #aaa; /* Sets the placeholder text color to light gray */
    opacity: 0.8; /* Makes the placeholder text slightly transparent */
}
</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: transparent; /* Makes the background transparent */
    color: #ffffff; /* Sets the text color */
    font-size: 12px; /* Sets the font size */
    font-weight: bold; /* Makes the font bold */
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow to the text */
    border: none; /* Ensures there is no border */
}
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background: transparent; /* Makes the background transparent */
    color: #ffffff; /* Sets the text color */
    font-size: 12px; /* Sets the font size */
    font-weight: bold; /* Makes the font bold */
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.1); /* Adds a subtle shadow to the text */
    border: none; /* Ensures there is no border */
}
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginInSignup">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>60</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: transparent; /* پس‌زمینه شفاف */
    color: rgb(0, 0, 0); /* رنگ متن اصلی */
    font-size: 10px; /* اندازه قلم */
    font-weight: bold; /* وزن قلم */
    border: none; /* بدون حاشیه */
    text-decoration: none; /* بدون تزئین متن */
    outline: none; /* بدون خط حاشیه دور روشن شدن */
    border-bottom: 2px solid black; /* خط زیری با رنگ سفید */
}

QPushButton::hover 
{
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>297</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameInLogIn</tabstop>
  <tabstop>PasswordInLogin</tabstop>
  <tabstop>loginBotton</tabstop>
  <tabstop>SignUpInLogIn</tabstop>
  <tabstop>usernameInSignup</tabstop>
  <tabstop>PasswordInSignup</tabstop>
  <tabstop>signupBotton</tabstop>
  <tabstop>LoginInSignup</tabstop>
  <tabstop>PasswordInLogin_2</tabstop>
  <tabstop>SignUpnInLogIn_2</tabstop>
  <tabstop>usernameInLogIn_2</tabstop>
  <tabstop>LoginButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="im.qrc"/>
 </resources>
 <connections/>
</ui>
